docs/architecture/porting-existing-aspnet-apps/choose-net-core-version.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Choosing the right .NET Core version
+description: How should you determine which version of .NET Core to target?
+author: ardalis
+ms.date: 11/13/2020
+---
+
+# Choose the right .NET Core version
+
+The largest consideration for most organizations when choosing a version of .NET Core to target is the support lifecycle. Long Term Support (LTS) releases ship less frequently but have a longer support window than Current (non-LTS) releases. Customers can choose which releases to target, and can install different releases of .NET Core side by side on the same machine. LTS releases will receive only critical and compatible fixes throughout their lifecycle. Current releases will receive these same fixes and will also be updated with compatible innovations and features. LTS releases are supported for three years after their initial release. Current releases are supported for three months after a subsequent Current or LTS release.
```

Comment:
> Long Term Support (LTS) releases ship less frequently but have a longer support window than Current (non-LTS) releases.

Should we clarify and say LTS releases ship every other year? Saying less frequently makes it appear it's a non-deterministic when we have a fairly well-defined schedule.